src/main/java/com/softserve/sprintone/luis/additionalResources/EncapsulationDemo.java

```diff
@@ -0,0 +1,16 @@
+package com.softserve.sprintone.luis.additionalResources;
+
+import lombok.AllArgsConstructor;
```

Comment:
good to know you are familiar with Lombok.

Reply:
I did not know lombok it was new for me, the task was about its use. 
https://www.codewars.com/kata/lombok-encapsulation